src/pages/SavedSearch/SavedSearchPage.tsx

```diff
@@ -74,6 +89,42 @@ const SavedSearchPage = (): JSX.Element => {
         };
     }, [params]);
 
+    const handleLoadMore = async (): Promise<void> => {
+        setFetchMoreResultsStatus(AsyncStatus.LOADING);
+        try {
+            const newResults = await api.getSavedSearchResults(
+                params.plant,
+                params.savedSearchId,
+                params.savedSearchType,
+                source.token,
+                nextPageNumber
+            );
+            if (newResults.length === 0) {
+                setFetchMoreResultsStatus(AsyncStatus.EMPTY_RESPONSE);
```

Comment:
Hei dere, det jeg kommer med nå tenker jeg ikke er nødvendig å ta i betrakting i denne pr'en , men ser det at slikt som fetchMoreResultsStatus er en gjennomgang i hele kodebasen. Her setter vi en `state` til et en `entry` i en `enum`. for så og sjekke opp `enum`. mulig dette burde være innebygd sjekker i apiClienten?